packages/pds-ember/addon/components/pds/page/stories/index.stories.js

```diff
@@ -0,0 +1,85 @@
+import hbs from 'htmlbars-inline-precompile'
+import DocsPage, { TITLE } from '../docs.mdx'
+
+export default {
+  title: TITLE,
+  component: 'PdsPage',
+  subcomponents: {
+    'P.Header': 'PdsPageHeader',
+    'P.Body': 'PdsPageBody',
+  },
+  parameters: { docs: { page: DocsPage } },
+  argTypes: {
+    headerVariant: {
+      name: 'header variant',
+      defaultValue: '',
+      control: {
+        type: 'radio',
+        options: {
+          'plain (default)': '',
+          'filled': 'filled',
+        },
+      },
+    },
+    headerActionVariant: {
+      name: 'header action variant',
+      control: {
+        type: 'radio',
+        options: ['primary', 'warning'],
+      }
+    },
+  },
+  args: {
+    showActions: true,
+    showTabs: true,
+    headerVariant: 'filled',
+    headerActionVariant: 'primary',
+    pageTitle: 'Page Title',
+  },
+}
+
+export const Index = (args) => ({
+  template: hbs`
+    <Pds::Page as |P|>
+      <P.Header @variant={{headerVariant}} as |H|>
+        <H.Breadcrumbs>
+          <Docs::Breadcrumbs
+            @depth={{3}}
+            @showIcon={{true}}
+          />
+        </H.Breadcrumbs>
+
+        <H.Title>
+          {{pageTitle}}
+        </H.Title>
+
+        {{#if showActions}}
+          <H.Actions>
+            <Pds::ButtonSet>
+              <Pds::Button @variant={{headerActionVariant}}>
+                {{headerActionVariant}}
```

Comment:
I went looking for this because I thought it was odd that an arg to Page would change the button variant, but I guess controls is less about direct args and more about showing all of the variants, docs are for args.